mlir/lib/Tools/PDLL/Parser/Lexer.cpp

```diff
@@ -295,6 +299,36 @@ void Lexer::lexComment() {
   }
 }
 
+/// Skip multi-line comments that start with /* and end with */.
+LogicalResult Lexer::lexBlockComment() {
+  // Advance over the '*' in a '/*' comment.
+  assert(*curPtr == '*');
+  ++curPtr;
+
+  while (true) {
+    switch (*curPtr++) {
+    case '*':
+      // Block ends with '*/'.
+      if (*curPtr++ == '/')
```

Comment:
If the block ends with `**/`, would this code find it? Please add a test for that case.